packages/manager/src/features/linodes/LinodesDetail/LinodeConfigs/DeleteConfigDialog.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Typography from 'src/components/core/Typography';
+import ActionsPanel from 'src/components/ActionsPanel';
+import Button from 'src/components/Button';
+import { Config } from '@linode/api-v4';
+import { useSnackbar } from 'notistack';
+import { ConfirmationDialog } from 'src/components/ConfirmationDialog/ConfirmationDialog';
+import { useLinodeConfigDeleteMutation } from 'src/queries/linodes/configs';
+
+interface Props {
+  open: boolean;
+  onClose: () => void;
+  linodeId: number;
+  config: Config | undefined;
+}
+
+export const DeleteConfigDialog = (props: Props) => {
+  const { open, onClose, linodeId, config } = props;
+  const { enqueueSnackbar } = useSnackbar();
+
+  const { mutateAsync, isLoading, error } = useLinodeConfigDeleteMutation(
+    linodeId,
+    config?.id ?? -1
+  );
+
+  const onDelete = async () => {
+    await mutateAsync();
+    enqueueSnackbar('Successfully deleted config', { variant: 'success' });
+    onClose();
+  };
+
+  const actions = (
+    <ActionsPanel style={{ padding: 0 }}>
+      <Button buttonType="secondary" onClick={onClose}>
+        Cancel
+      </Button>
+      <Button buttonType="primary" onClick={onDelete} loading={isLoading}>
+        Delete
+      </Button>
+    </ActionsPanel>
+  );
+
+  return (
+    <ConfirmationDialog
+      title="Confirm Delete"
+      error={error?.[0].reason}
+      open={open}
+      onClose={onClose}
+      actions={actions}
+    >
+      <Typography>
+        Are you sure you want to delete &quot;
+        {config?.label}?&quot;
```

Comment:
This is also a minor typo in prod, but we could fix it here! The question mark comes before the closing quote instead of after:
![Screenshot 2023-06-05 at 9 28 58 AM](https://github.com/linode/manager/assets/114685994/5e6b4edb-e772-488e-ae2f-e5019eda72f4)

Also noticed that the "Cancel" button doesn't seem to be working in prod, but is fixed on this branch. 🎉 
```suggestion
        {config?.label}&quot;?
```